TS.fsx

```diff
@@ -381,8 +381,15 @@ let EmitNamedConstructors () =
 
 let EmitInterfaceDeclaration (i:Browser.Interface) =
```

Comment:
Looks like this only works with interfaces. Is there a need for classes too? (I'm not familiar with lib.d.ts so could be there's not.)